flexget/plugins/input/trakt_emit.py

```diff
@@ -33,26 +34,29 @@ class TraktEmit(object):
         'type': 'object',
         'properties': {
             'username': {'type': 'string'},
-            'password': {'type': 'string'},
+            'account': {'type': 'string'},
             'position': {'type': 'string', 'enum': ['last', 'next'], 'default': 'next'},
             'context': {'type': 'string', 'enum': ['watched', 'collected'], 'default': 'watched'},
             'list': {'type': 'string'}
         },
-        'required': ['username'],
+        'anyOf': [{'required': ['username']}, {'required': ['account']}],
+        'error_anyOf': 'At least one of `username` or `account` options are needed.',
         'additionalProperties': False
     }
 
     def on_task_input(self, task, config):
-        session = get_session(config['username'], config.get('password'))
+        if config.get('account') and not config.get('username'):
```

Comment:
Yeah, I'm not fully sure how I feel about defaults being schema only. Even if we do it through code I guess it could just `config.setdefault('username', 'me')` without the conditional.